AMWin-RichPresence/SettingsWindow.xaml

```diff
@@ -62,8 +60,10 @@
                         <TextBox Grid.Row="6" Grid.Column="1" x:Name="LastfmUsername" Text="{Binding Source={x:Static properties:Settings.Default}, Path=Default.LastfmUsername}" TextWrapping="Wrap" Margin="0,0,-121,0" VerticalAlignment="Center"/>
                         <TextBlock Grid.Row="7" Grid.Column="0" VerticalAlignment="Center" Padding="10 0 0 0">Password</TextBlock>
                         <PasswordBox Grid.Row="7" Grid.Column="1" x:Name="LastfmPassword" VerticalAlignment="Center" Margin="0,0,-121,0"/>
-                        <Button Grid.Row="8" Grid.Column="1" Content="Button" HorizontalAlignment="Left" Margin="201,10,0,0" VerticalAlignment="Top" Width="0"/>
-                        <Button Grid.Row="8" Grid.Column="1" x:Name="SaveLastFMCreds" Content="Save Last.FM Credentials" VerticalAlignment="Center" Margin="99,0,-121,0" Click="SaveLastFMCreds_Click"/>
+                        <TextBlock Grid.Row="8" Grid.Column="0" Margin="10,2,17,3" Grid.ColumnSpan="2">Scrobble only Primary Artist (EXPERIMENTAL)</TextBlock>
```

Comment:
We don't have to put an experimental tag on here - it should be enough to just mention it in the release notes.

Reply:
@cillyk